macros/schema_tests/equal_sum.sql

```diff
@@ -0,0 +1,34 @@
+{% macro test_equal_sum(model) %}
+
+{% set column_name = kwargs.get('column_name', kwargs.get('arg')) %}
+{% set compare_model = kwargs.get('compare_model', kwargs.get('arg')) %}
+{% set compare_column_name = kwargs.get('compare_column', kwargs.get('arg')) %}
+
+{#-- Prevent querying of db in parsing mode. This works because this macro does not create any new refs. #}
+{%- if not execute -%}
+    {{ return('') }}
+{% endif %}
+
+with a as (
+
+    select sum({{ column_name }}) as sum_a from {{ model }}
+
+),
+b as (
+
+    select sum({{ compare_column_name }}) as sum_b from {{ compare_model }}
```

Comment:
make sure to change this to `compare_column` if you change the macro signature above